modules/tinymce/src/core/test/ts/browser/html/DomParserTest.ts

```diff
@@ -1665,12 +1665,13 @@ describe('browser.tinymce.core.html.DomParserTest', () => {
       assert.equal(serializedHtml, '<svg></svg><p>foo</p>');
     });
 
+    // Updated for TINY-11332: Remove html elements inside SVG
```

Comment:
It would be good to also add another test case `it` block to this file which checks some other elements are not removed e.g.

```
  it('TINY-11332: Should retain SVG elements and keep HTML elements that are valid inside an SVG', () => {
    const schema = Schema();
    schema.addValidElements('svg[*]');
    const input = '<svg><a href="/docs/Web/SVG/Element/circle"><circle cx="50" cy="40" r="35" /></a><script>alert(1)</script></svg>foo';
    const serializedHtml = HtmlSerializer({}, schema).serialize(DomParser({ forced_root_block: 'p' }, schema).parse(input));
    assert.equal(serializedHtml, '<svg><a href="/docs/Web/SVG/Element/circle"><circle cx="50" cy="40" r="35"></circle></a></svg><p>foo</p>');
  });
```